conman/routes/tests/test_handlers.py

```diff
@@ -34,6 +20,19 @@ def test_init(self):
         self.assertEqual(handler.route, route)
 
 
+class BaseHandlerCheckTest(TestCase):
+    """Test BaseHandler.check()."""
+    def test_classmethod(self):
+        """Ensure check() is a classmethod."""
+        self.assertIsInstance(vars(BaseHandler)['check'], classmethod)
```

Comment:
Can we use `BaseHander.check` instead of `vars(BaseHandler)['check']`? Otherwise a comment.